### DISCUSSION TITLE
16X2 lcd displays every other letter

### DISCUSSION THREAD

**Discussion by AlexArtCie** [probonopd/MiniDexed#546] [Q&A] [answered]
hello, I don't understand why my 16X2 lcd displays every other letter, with a rasberry 3B 1 2. with the latest version. Does anyone have an answer thank you

**Comment by Banana71** [comment]
Hi Alex,
This is a bug that we haven't had yet.  Can you post a photo of it and show us your minidexed.ini.  Do you have an i2c backpack board?

**Comment by AlexArtCie** [comment]
Hi ,
thank you for your reply. yes I have a 16x2 i2c LCD but for 5 volts I will see a 3.3v converter, normally I have one. (is the converter really useful?) Have you the picture?
![1695886459707](https://github.com/probonopd/MiniDexed/assets/146321390/a16aa789-3dff-49fc-9ac9-20b92dcdb38b)

**Reply by Banana71** [reply]
Are the data lines from the display really wired correctly to the Raspberry? D4-D7 on pins 15, 16, 18, 22. The yellow line looks very suspicious. Does D4 go to pin 15?

**Comment by AlexArtCie** [comment]
ok, my i2c lcd displays nothing its address is OX3F

**Comment by AlexArtCie** [comment]
I looked in the other questions answers, I found how to change the address of the i2c lcd in the minidexed file.... it's OK. 
thanks. Regards

**Comment by AlexArtCie** [comment, marked as answer]
I had this Raspberry with crooked pins... I straightened them... the problem must certainly come from there. Thank you for this discussion, I succeeded with the i2c, it's great. Thanks again

**Reply by Banana71** [reply]
Have fun with the miniDexed. :-)
The difference with a DAC is huge.

**Comment by AlexArtCie** [comment]
yes I know, an acquaintance already made me hear it.
it's like a DX7 but even better quality,
and then the filter, the reverb... it's great!!!
I ordered it and should receive it very soon.
Thank you again for this quick discussion.
for information, I am in France
